recipe-client-addon/lib/RecipeRunner.jsm

```diff
@@ -66,7 +73,24 @@ this.RecipeRunner = {
     return true;
   },
 
-  async start() {
+  observe(changedPrefBranch, action, changedPref) {
```

Comment:
A short doc comment would be nice. I missed at first that `RecipeRunner` itself was the preference observer.